test/unit/typings-compatibility-test.ts

```diff
@@ -101,7 +73,7 @@ async function run(
       if (error) {
         resolve(errorAsString ? stdout + "\n" + stderr : error)
       } else {
-        resolve()
+        resolve(undefined)
```

Comment:
Why is this necessary?

Reply:
Resolve always expects value. Not passing anything is equal to passing undefined. In the recent versions of TypeScript, it errors when you call resolve without any arguments. 
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/resolve